src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -540,6 +541,21 @@ internal override CSharpAttributeData EarlyDecodeWellKnownAttribute(ref EarlyDec
                 return null;
             }
 
+            if (CSharpAttributeData.IsTargetEarlyAttribute(arguments.AttributeType, arguments.AttributeSyntax, AttributeDescription.CodeAnalysisEmbeddedAttribute))
```

Comment:
Discussed offline. This is following VB.NET EmbeddedAttribute.

---
In reply to: [113562877](https://github.com/dotnet/roslyn/pull/18715#discussion_r113562877) [](ancestors = 113562877)